app/src/main/res/layout/widget_button_configure.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:padding="16dp">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
+        android:text="@string/configure_service_call" />
+
+    <EditText
+        android:id="@+id/widget_text_config_domain"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/widget_text_hint_service_domain"
+        android:inputType="text" />
+
+    <EditText
+        android:id="@+id/widget_text_config_service"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/widget_text_hint_service_service"
+        android:inputType="text" />
+
+    <EditText
+        android:id="@+id/widget_text_config_service_data"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/widget_text_hint_service_data"
+        android:inputType="text" />
```

Comment:
I think we should have labels for all these.

Reply:
I have changed these to the TextInputEditText layout and changed the labels from the lowercase JSON-type format to a regular text label format.

Screenshot here:
![Screenshot_20200117-121930_Home Assistant](https://user-images.githubusercontent.com/8548596/72632699-d302bf80-3924-11ea-83a6-94ae190debba.jpg)